### DISCUSSION TITLE
请求增加分类同步功能，或者多设备同步功能。

### DISCUSSION THREAD

**Discussion by yw124** [Planshit/Tai#207] [Ideas]
首先非常感觉作者的软件，让自己意识到虽然每天都在电脑前，但实际上工作时间非常短，提醒自己合理利用时间。
其次自己因为实际需要，经常需要使用多设备，在台式机和笔记本电脑转换，导致时间记录和分类标签记录不连贯。
因此恳求作者有空开发一个能基于坚果云或者onedrive的功能，可以将某台设备的分类标签与统计数据导入到其他设备当中。
非常感谢！

**Comment by DaTouJun** [comment]
或者基于Amazon S3 + 文件的方法也好
